CustomerApp/src/main/java/com/brainscratch/laboratory/customerapp/Json.java

```diff
@@ -0,0 +1,45 @@
+package com.brainscratch.laboratory.customerapp;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.IOException;
+
```

Comment:
So you need to create CustomerFile or CustomerFileProcessor and describe methods marshal\demarshal JSON files for customers.

Reply:
okay i need to research this because i'm not familiar